### ISSUE TITLE
Rich text support.

### ISSUE THREAD

**Issue by michaelthatsit** [Volumetrics-io/mrjs#340]
This is a catch all for text. We need:

- nested text (i.e. `<mr-text> parent <mr-text> child </mr-text><mr-text>`)
- nested styling (i.e. `<mr-text> parent <mr-text style="color: blue"> child </mr-text><mr-text>`)
- advanced styling (i.e. video textures, variable fonts, essentially support for all typography related CSS properties)
- ?? @lobau looking to you for more details.

**Comment by michaelthatsit** [comment]
Yeah there are a couple that relate to this. I'll see if I can dig them up

**Comment by lobau** [comment]
As input, I would say that I think we should support the basic formating tags like `<em>`, `<strong>`, `<i>`, `<b>`, etc.
As if you want to bold a word, you really don't want to have to go create a CSS class for this and do something like

```html
<mr-text>This is a <mr-text class="bold">bold</mr-text> word.</mr-text>
```

I would rather do this:


```html
<mr-text>This is a <b>bold</b> word.</mr-text>
```

**Comment by michaelthatsit** [comment]
Yeah this is valid. Ideally we could replace `mr-text` with any standard HTML tags within an `mr-panel` or `mr-div`
